README.md

```diff
@@ -180,12 +180,14 @@ Usage instructions for each are below.
 ```
 cmake . && make
 ```
-Note that you can also use `ninja` as your generator: `cmake -G Ninja . && ninja`
 
-* A C++11 compiler is required.
-* The JavaScript components can be built using `build-js.sh`, see notes inside. Normally this is not needed as builds are provided in this repo already.
+A C++14 compiler is required. Note that you can also use `ninja` as your generator: `cmake -G Ninja . && ninja`.
```

Comment:
Interesting, when did the change to c++14 happen?

Reply:
#2358, after briefly trying to go straight to C++17